src/App.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+import TodoPage from './TodoPage';
+
+function App() {
+  const [state, setState] = useState({
+    newTodo: {
+      id: 1,
+      content: '',
+    },
+    todos: [],
+  });
```

Comment:
예시를 들었던 거였는데, 제가 설명이 부족했던것 같네요~

```suggestion
    newTodo: '',
    todos: [
      {
        id: 1,
        content: ''
      }
    ],
  });
```
지금 구조와 비슷하지만, 조금 구조를 바꿔서 이렇게 되면 어떤가요?

Reply:
제 생각엔 해당 구조로 바꾸면 좋은 이유가

1. newTodo값이 변경될때 id를 유지 하지 않아도 된다.
2. id값은 todos 내의 todo의 고유번호를 매기기 위해 필요하기 때문에??

이 정도로 생각했는데.. 다른 이유가 있을까요?

Reply:
ㅎㅎ 맞습니다 ☺️ newTodo에는 새로 입력 받은 값만 받도록 해서 조금 더 명확하고, newTodo를 todos에 넣어줄때 id값을 지정해줄 수 있을 것 같네요~